packages/help-center/src/components/help-center-content.tsx

```diff
@@ -75,13 +75,20 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 		} );
 	}, [ location, sectionName ] );
 
-	const { initialRoute } = useSelect(
+	const { navigateToRoute } = useSelect(
 		( select ) => ( {
-			initialRoute: ( select( HELP_CENTER_STORE ) as HelpCenterSelect ).getInitialRoute(),
+			navigateToRoute: ( select( HELP_CENTER_STORE ) as HelpCenterSelect ).getNavigateToRoute(),
 		} ),
 		[]
 	);
 
+	useEffect( () => {
+		if ( navigateToRoute ) {
+			navigate( navigateToRoute );
```

Comment:
I recently removed the use of `navigate()` on the initial load because it made it impossible to determine if it was the beginning of the history or not. When the route is first loaded the `location.key` is set to `default`. When you `navigate()` to another path the `location.key` updates to a random string. This is how the back button determines if it has the ability to go backwards. To ensure the `key` is not set when the initial route was set we had to use the `<Navigate />` below on line 120.

Testing this PR doesn't immediately show the issue because there is another bug here ⇢ https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/components/help-center-article.tsx#L64 That line should be reversed `!==`.

Reply:
Adding notes for myself:

Inverse L64 in `help-center-article` Line 64 (`if ( key === 'default' )`).
If the key is not default, then we know there is a navigation history.
Figure out how to refresh/update the route without including the `navigate( navigateToRoute )` hook in the useEffect (`help-center-content`).